react/features/base/premeeting/functions.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+const ratio = 1 / 3;
+const maxSize = 190;
+const upperHeightLimit = 760;
+const lowerHeightLimit = 460;
+const defaultMarginTop = '10%';
+const smallMarginTop = '5%';
+
+/**
+ * Calculates avatar dimensions based on window height and position.
+ *
+ * @param {Object} window - The window object.
+ * @returns {{
+ *   marginTop: string,
+ *   size: number
+ * }}
+ */
+export function calculateAvatarDimensions(window: Object) {
+    const height = window.innerHeight;
```

Comment:
Actually in features/base/responsive-ui we have `clientResized` and we store exactly the inner width/height in redux. Maybe you can reuse those. This will also probably simplify the code in the component because you won't need to add/remove listeners for resize. WDYT?

Reply:
Great catch! Did not know about that. Used the hight from the store.